capstone/src/main/webapp/script.js

```diff
@@ -294,6 +303,33 @@ function showAvailableAttributes() {
  * Highlight the currently selected choice(s) in dropdown menus
  */
 function loadDropdowns() {
+  // Create language options
+  const availableLanguagesElement = document.getElementById('available-languages');
+  if (availableLanguagesElement == null) {
+    return;
+  }
+  availableLanguagesElement.innerHTML = '';
+	
+  for (const lang of Object.keys(LANGUAGES)) {
+    const radio = document.createElement('input');
+    radio.name = 'languageRadios';
+    radio.type = 'radio';
+    radio.value = lang;
+  
+    const label = document.createElement('label');
+    label.appendChild(radio);
+    label.innerHTML += LANGUAGES[lang];
+
+    const list = document.createElement('li');
+    // list.className = 'active';
```

Comment:
nit: Let's delete this commented-out code

Reply:
Fixed